core/src/main/java/org/infinispan/configuration/cache/ClusteringConfiguration.java

```diff
@@ -19,7 +19,7 @@
          AttributeDefinition.builder("remoteTimeout", TimeUnit.SECONDS.toMillis(15)).build();
 
    static AttributeSet attributeDefinitionSet() {
-      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE);
+      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE, REMOTE_TIMEOUT);
```

Comment:
Since it is ClusteringConfiguration which owns the attribute, it makes sense for it to exist in that attribute set rather than in the deprecated one. AttributeSets matter when copying a configuration and when computing equality, otherwise it doesn't matter much.